contrib/borg/template.py

```diff
@@ -0,0 +1,54 @@
+pkgname = "borg"
+pkgver = "1.2.8"
+pkgrel = 0
+build_style = "python_pep517"
+make_build_env = {
+    "BORG_LIBLZ4_PREFIX": "/usr/include",
+    "BORG_ZSTD_PREFIX": "/usr/include",
+    "BORG_XXHASH_PREFIX": "/usr/include",
+}
+make_check_args = ["--pyargs", "borg.testsuite"]
+hostmakedepends = [
+    "pkgconf",
+    "python-build",
+    "python-cython",
+    "python-installer",
+    "python-pkgconfig",
+    "python-setuptools",
+    "python-setuptools_scm",
+    "python-wheel",
+]
+makedepends = [
+    "acl-devel",
+    "linux-headers",
+    "lz4-devel",
+    "openssl-devel",
+    "python-devel",
+    "xxhash-devel",
+    "zstd-devel",
+]
+depends = ["python-packaging", "python-msgpack", "python-pyfuse"]
+checkdepends = [
+    "python-pytest",
+    "python-dateutil",
+    "python-pytest-xdist",
+] + depends
+pkgdesc = "Deduplicating backup program"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "BSD-3-Clause"
+url = "https://www.borgbackup.org"
+source = f"https://github.com/borgbackup/borg/releases/download/{pkgver}/borgbackup-{pkgver}.tar.gz"
+sha256 = "d39d22b0d2cb745584d68608a179b6c75f7b40e496e96feb789e41d34991f4aa"
+# takes forever
+options = ["!check"]
```

Comment:
let's see how long

also might need BORG_FUSE_IMPL='none' in check_env and also pass --benchmark-skip in check_args